test-suite/Math/NumberTheory/GaussianIntegersTests.hs

```diff
@@ -14,22 +14,35 @@ module Math.NumberTheory.GaussianIntegersTests
   ( testSuite
   ) where
 
+import Control.Monad (zipWithM_)
 import Test.Tasty
 import Test.Tasty.HUnit
 
 import Math.NumberTheory.GaussianIntegers
 import Math.NumberTheory.TestUtils
 
+lazyCases :: [(GaussianInteger, [(GaussianInteger, Int)])]
+lazyCases =
+  [ ( 14145130710
```

Comment:
It is better to use big prime p = 4k+1, which cannot be factored by trial division, instead of an even number. For instance, 14145130733 will do. It has non-trivial factors 117058±21037*ι.

Reply:
Done.